source/backups.rst

```diff
@@ -178,7 +179,12 @@ Following steps are needed to restore a previously saved backup:
 Delete the unneeded backup
 --------------------------
 
-Deleting a previously saved backup requires not more than the backup
+The maximum amount of stored backups is controlled by the
+:ref:`backup.tasks.keep<backup-tasks-keep>` option. Older backups are
+automatically deleted, so that amount of stored backups do not exceed this
+number.
```

Comment:
I think we should mentioned that this is only true for successful backups. If backup has been finished with error we do not delete it.